R/ResampleDesc.R

```diff
@@ -47,6 +48,13 @@
 #'     Default is 10.}
 #'   \item{folds [\code{integer(1)]}}{Folds in the repeated CV for \code{RepCV}.
 #'     Here \code{iters = folds * reps}. Default is 10.}
+#'   \item{horizon [\code{integer(1)}]}{Number of observations to forecast for \dQuote{GrowingWindowCV}
+#'    and \dQuote{FixedWindowCV}. Default is 1.}
+#'   \item{initial.window [\code{numeric(1)}]}{Fraction of observations to start with
+#'    in \dQuote{GrowingWindowCV} and \dQuote{FixedWindowCV}. Default is 0.5.}
+#'   \item{skip [\code{integer(1)}]}{ How many resamples to skip to thin the total amount
+#'    for \dQuote{GrowingWindowCV} and \dQuote{FixedWindowCV}. This is passed through as the \dQuote{by} argument
+#'    in \code{seq()}. Default is horizon which gives mutually exclusive chunks of test indices.}
```

Comment:
This seems to be a bit inconsistent to me -- `horizon` and `skip` are numbers of observations, `initial.window` is a fraction. Could we support both in all? I.e. < 1 means fraction, >= 1 number of observations.

Reply:
Yes the more I thought about this the more I liked it! rn I have
If value < 1:
skip is scale by the length the resampling index
initial.window is scaled by size
horizon is scaled by the length of the initial window

Are those sensible?